assets/stylesheets/shared/functions/_z-index.scss

```diff
@@ -189,6 +189,9 @@ $z-layers: (
 		'.following-edit__subscribe-form .card.is-search-result': 35,
 		'.following-edit__subscribe-form .search': 36
 	),
+	'.gravatar-updater__gravatar-container': (
+		'.gravatar-updater__spinner': 1
+	),
```

Comment:
z-index are a bit complicated, but the parent stacking context in this case is root:
<img width="1231" alt="screen shot 2016-10-07 at 2 06 51 pm" src="https://cloud.githubusercontent.com/assets/1270189/19205476/945f7e04-8c97-11e6-910c-b33e9f028d55.png">

```
'root': (
//...
    '.gravatar-updater__spinner': 1,
```

See also: https://github.com/gwwar/z-context for help on looking up this info